prow/jobs/custom/test-infra.yaml

```diff
@@ -154,32 +135,6 @@ periodics:
         requests:
           cpu: 100m
           memory: 1Gi
-- cron: "15 9 * * *"
-  name: ci-knative-backup-artifacts
```

Comment:
Was it intentional to remove this job?

Reply:
ah, seems like unit tests caught this was missing. If it is an intentional removal then looks like unit tests will need to be updated as well.

Reply:
Yes, it has been failing for a long while - https://prow.knative.dev/?job=ci-knative-backup-artifacts